gobblin-data-management/src/test/java/org/apache/gobblin/data/management/copy/iceberg/IcebergDatasetTest.java

```diff
@@ -187,7 +188,7 @@ public void testGetFilePathsDoesNotSwallowDestFileSystemException() throws IOExc
 
     MockFileSystemBuilder sourceFsBuilder = new MockFileSystemBuilder(SRC_FS_URI);
     FileSystem sourceFs = sourceFsBuilder.build();
-    IcebergDataset icebergDataset = new IcebergDataset(testDbName, testTblName, icebergTable, new Properties(), sourceFs);
+    IcebergDataset icebergDataset = new IcebergDataset(testDbName, testTblName, icebergTable, SRC_CATALOG_URI, new Properties(), sourceFs);
```

Comment:
in the suggestion above, this additional new param goes away

Reply:
true... made the changes accordingly